spec/migrations/20160127210624_convert_configurations_to_settings_changes_spec.rb

```diff
@@ -57,8 +57,11 @@
         "vmdb"   => YAML.load_file(data_dir.join("simple.tmpl.yml")).deep_symbolize_keys
       }
 
+      # capture stdout, because stub_const will trigger say_with_time
+      $stdout = StringIO.new
```

Comment:
without this I get
```
❯ be spring rspec spec/migrations/20160127210624_convert_configurations_to_settings_changes_spec.rb
Running via Spring preloader in process 46228

Randomized with seed 45098
..-- to_str()
-- to_str()
-- to_ary()
.

Finished in 0.28894 seconds (files took 1.98 seconds to load)
3 examples, 0 failures
```

which is the output from [say_with_time](https://github.com/rails/rails/blob/master/activerecord/lib/active_record/migration.rb#L816)
unfortunately I dont know why - and I would not really care, because spec are green and I guess its not `worth_the_time` 😄